Go_Refined_Code/database/db_migrate.py

```
@@ -0,0 +1,123 @@
+    cols_quoted = ", ".join(f'"{col}"' for col in columns)
+
+    batch.append(tuple(first_row.values()))
+
+    for row in mysql_cursor:
+        batch.append(tuple(row.values()))
+
+        if len(batch) >= BATCH_SIZE:
+            try:
+                execute_values(
+                    pg_cursor,
+                    f'INSERT INTO "{table}" ({cols_quoted}) VALUES %s ON CONFLICT DO NOTHING',
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)